ui/components/app/connected-sites-list/connected-sites-list.component.js

```diff
@@ -39,11 +39,12 @@ export default class ConnectedSitesList extends Component {
                 {this.getSubjectDisplayName(subject)}
               </span>
             </div>
-            <i
-              className="fas fa-trash-alt connected-sites-list__trash"
-              title={t('disconnect')}
+            <div
+              className="connected-sites-list__disconnect"
               onClick={() => onDisconnect(subject.origin)}
-            />
+            >
+              {t('disconnect')}
```

Comment:
We could split out this change

Reply:
Though also we shouldn't make clickable divs. This is better off as a button.

Reply:
Updated in #13381

We could perhaps still split it out @shanejonas